clang/test/Interpreter/lit.local.cfg

```diff
@@ -1,2 +1,5 @@
-if "host-supports-jit" not in config.available_features:
+import sys
+
+# clang-repl is not supported on AIX and zOS
+if "host-supports-jit" not in config.available_features or sys.platform == "aix" or sys.platform == "zos":
```

Comment:
I think it amounts to the same thing, but since we have lit features for these things already, we should use them. For instance file elsewhere does:
```
if "system-aix" in config.available_features:
    config.unsupported = True
```
Though that is in `llvm` and the feature is added by llvm/utils/lit/lit/llvm/config.py. You might have to add the zOS feature to clang's equivalent file.